### PR TITLE
Display outbound messages on identity page

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
ci_outbounds

File: ci/utils.py [hunk 1]
```diff
@@ -101,3 +102,9 @@ def get_ranged_data_from_timeseries(timeseries, dt, range_type='week'):
     end = get_timestamp(boundries.end)
     sent_data = transform_timeseries_data(timeseries, start, end)
     return right_pad_list(sent_data, length=padding, value=0)
+
+
+def extract_query_params(url_str):
+    if url_str is None:
```
Comment (rudigiesler) [inline ci/utils.py]: Would it not be better here to ensure that it's a string type, rather than just `None`?

**Review by rudigiesler — COMMENTED** [review]
Just the one comment from me

File: ci/utils.py (hunk 1 above)
Comment (KaitCrawford) [reply ci/utils.py]: yeah, I was going to just steal your `extract_query_params` from the inbound PR but I figured you'd have to choose between them when you merged anyway. But that's what I've done now :P

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**